### PR TITLE
Ultra.Ultra version 1.0.0.4

### PR DESCRIPTION
**Pull request by Sky-NiniKo**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/56612)

### CODE REVIEW

File: manifests/u/Ultra/Ultra/1.0.0.4/Ultra.Ultra.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Ultra.Ultra
+PackageVersion: 1.0.0.4
+PackageLocale: en-US
+Publisher: Ultra
```
Comment (Trenly) [inline manifests/u/Ultra/Ultra/1.0.0.4/Ultra.Ultra.locale.en-US.yaml:7]:
```suggestion
Publisher: The Ultra Authors
```
This must match Control Panel

File: manifests/u/Ultra/Ultra/1.0.0.4/Ultra.Ultra.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Ultra.Ultra
+PackageVersion: 1.0.0.4
```
Comment (Trenly) [inline manifests/u/Ultra/Ultra/1.0.0.4/Ultra.Ultra.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 5.0.5.0
```
This must match the version in control panel

File: manifests/u/Ultra/Ultra/1.0.0.4/Ultra.Ultra.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Ultra.Ultra
+PackageVersion: 1.0.0.4
```
Comment (Trenly) [inline manifests/u/Ultra/Ultra/1.0.0.4/Ultra.Ultra.installer.yaml:5]:
```suggestion
PackageVersion: 5.0.5.0
```
This must match the version in control panel

File: manifests/u/Ultra/Ultra/1.0.0.4/Ultra.Ultra.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: Ultra.Ultra
+PackageVersion: 1.0.0.4
```
Comment (Trenly) [inline manifests/u/Ultra/Ultra/1.0.0.4/Ultra.Ultra.yaml:5]:
```suggestion
PackageVersion: 5.0.5.0
```
This must match the version in control panel

File: manifests/u/Ultra/Ultra/1.0.0.4/Ultra.Ultra.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Ultra.Ultra
+PackageVersion: 1.0.0.4
+PackageLocale: en-US
+Publisher: Ultra
+PackageName: Ultra
+License: Proprietary
+ShortDescription: Ultra Installer
```
Comment (Trenly) [inline manifests/u/Ultra/Ultra/1.0.0.4/Ultra.Ultra.locale.en-US.yaml:10]: Please provide a better description of what `Ultra` is

**Review by Trenly — CHANGES_REQUESTED** [review]
When changing the `PackageVersion` you will also need to update the name of the version number folder

File: manifests/u/Ultra/Ultra/1.0.0.4/Ultra.Ultra.locale.en-US.yaml (hunk 5 above)
Comment (Sky-NiniKo) [reply manifests/u/Ultra/Ultra/1.0.0.4/Ultra.Ultra.locale.en-US.yaml]: Fix by commit [5a12ab4](https://github.com/microsoft/winget-pkgs/pull/56612/commits/5a12ab41d776f7bfef3f59bb2cf134423b24ea9c)

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**